src/math.F90

```diff
@@ -659,4 +675,91 @@ function watt_spectrum(a, b) result(E_out)
 
   end function watt_spectrum
 
+!===============================================================================
+! W acts as a front end to the MIT Faddeeva function, Faddeeva_w.
+!===============================================================================
+
+  function w(z) result(wv)
```

Comment:
The name `w` is not very descriptive.  I would recommend replacing this with `faddeeva`